INSTALL

```diff
@@ -17,13 +17,23 @@ Getting Subsurface source
 -------------------------
 
 You can get the sources to the latest development version from our git
-repository http://github.com/Subsurface-divelog/subsurface.git
-
-Remark, if you plan on making pull requests, the best way is to clone
-subsurface.git to your GitHub account, make a new branch (e.g. myPatches).
-Then clone that to your local computer. Doing this makes it very simple
-to generate pull requests.
-
+repository:
+    git clone http://github.com/Subsurface-divelog/subsurface.git
+
+You keep it updated by doing:
+    git checkout master
+    git pull -r
+    
+If you plan on making pull requests, you need furthermore to:
+   - clone subsurface.git to your GitHub account
```

Comment:
the wording that GitHub uses is "fork the subsurface repository from your GitHub account" - we should match that.